__tests__/__integration__/imperative/package.json

```diff
@@ -45,7 +45,7 @@
     "js-yaml": "3.13.1",
     "jsonfile": "4.0.0",
     "jsonschema": "1.1.1",
-    "keytar": "7.1.0",
+    "keytar": "6.0.1",
```

Comment:
Curious why the downgrade of Keytar package?

Reply:
likely a mistake on my part - ill change this